accelerator/cpu_accelerator.py

```diff
@@ -320,3 +320,10 @@ def build_extension(self):
 
     def export_envs(self):
         return []
+
+    def visible_devices_envs(self):
+        return ['CUDA_VISIBLE_DEVICES']
```

Comment:
@delock, I suspect that `CUDA_VISIBLE_DEVICES` is not correct for cpu_accelerator?